### PR TITLE
Fix null missing check

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-top-scroll-null-check](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-top-scroll-null-check)

File: src/app/public/modules/grid/grid.component.ts [hunk 1]
```diff
@@ -598,22 +598,28 @@ export class SkyGridComponent implements OnInit, AfterContentInit, OnChanges, On
...
 
   public onTopScroll(event: any): void {
-    if (this.scrollTriggered) {
-      this.scrollTriggered = false;
-      this.tableContainerElementRef.nativeElement.scrollLeft =
-        this.topScrollContainerElementRef.nativeElement.scrollLeft;
-    } else {
-      this.scrollTriggered = true;
+    /* sanity check */
+    if (this.tableContainerElementRef) {
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/grid/grid.component.ts]: Will this condition drop our coverage?

Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/grid/grid.component.ts]: It didn't drop anything but our branch coverage. I've added an istanbul flag to fix that too

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-top-scroll-null-check](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-top-scroll-null-check)

**Pull request merged**